testnets/qubeticsstestnet/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "qubeticstestnet",
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Qubetics Chain",
+      "denom_units": [
+        {
+          "denom": "tics",
+          "exponent": 0
+        },
+        {
+          "denom": "tics",
+          "exponent": 18
+        }
+      ],
+      "base": "tics",
+      "name": "Qubetics Testnet",
+      "display": "tics",
+      "symbol": "TICS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/qubeticstestnet/images/qubetics.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/qubeticstestnet/images/qubetics.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/qubeticstestnet/images/qubetics.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/qubeticstestnet/images/qubetics.svg"
```

Comment:
Please remove the svg. It looks like a bad png to svg conversion just for the sake of having an svg